traffic-policy/actions/forward-internal/examples/basic-example.mdx

```diff
@@ -0,0 +1,65 @@
+import ConfigExample from "/src/components/ConfigExample.tsx";
+
+### Basic Example
+
+This example configuration will set up a public endpoin (`forward-internal-example.ngrok.io`) which will forward all traffic it receives to a private endpoint (`example.private`) which will then forward the request to port `80` on your local machine. Since it is forwarding all traffic to the private endpoint, no traffic will be sent to `8080` which is the upstream port for the public endpoint.
```

Comment:
```suggestion
This example configuration will set up a public endpoint (`forward-internal-example.ngrok.io`) forwarding all traffic it receives to a private endpoint (`example.private`) that forwards the request to port `80` on your local machine. Since it is forwarding all traffic to the private endpoint, no traffic will be sent to `8080` which is the upstream port for the public endpoint.
```